src/fixes/unordered_key.rs

```diff
@@ -61,11 +62,38 @@ impl Fix for UnorderedKeyFixer<'_> {
                     end.replace(i + 1);
                 }
 
-                if line.is_empty() || line.is_last_line() || is_control_comment {
+                let subst_keys = line.get_substitution_keys();
+                let mut replaced_line = None;
+                for j in start_index..i {
+                    let key = match lines[j].get_key() {
+                        Some(k) => k,
+                        None => continue,
+                    };
+
+                    if subst_keys.iter().any(|k| k == &key) {
+                        replaced_line = Some(format!(
+                            "{} # Unordered, {} uses {}",
+                            &line.raw_string,
+                            &lines[i].get_key().unwrap(),
```

Comment:
`unwrap` can cause the panic

Reply:
Wow, I didn't know that the `?` operator returns `None` in functions that return an `Option`. That's cool. I worry though that returning `None` here is a form of silently failing, and we would rather inform the user that we've reached a program state that we expected to never reach.

Reply:
That is good point 👍 
I think it can be done at upper level by comparing number of warnings and expected number of fixes and wrap that to `Result` with custom error.

However I suppose it is worth considering in a separate issue